packages/prosess-beregningsgrunnlag/src/components/ytelsesspesifikkseOpplysninger/frisinn/BeregningsresultatPeriode.jsx

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
+import moment from 'moment';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Column, Row } from 'nav-frontend-grid';
+import { Element, Normaltekst } from 'nav-frontend-typografi';
+import aktivitetStatus from '@fpsak-frontend/kodeverk/src/aktivitetStatus';
+import { DDMMYYYY_DATE_FORMAT, formatCurrencyNoKr } from '@fpsak-frontend/utils';
+import { VerticalSpacer } from '@fpsak-frontend/shared-components';
+import beregningStyles from '../../beregningsgrunnlagPanel/beregningsgrunnlag.less';
+
+const finnSamletBruttoForStatus = (andeler, status) => {
+  if (!andeler) {
+    return 0;
+  }
+  return andeler
+    .filter(a => a.aktivitetStatus.kode === status)
+    .map(({ bruttoPrAar }) => bruttoPrAar)
+    .reduce((sum, brutto) => sum + brutto, 0);
+};
+
+const lagPeriodeHeader = (fom, originalTom) => {
+  let tom = null;
+  if (originalTom !== '9999-12-31') {
+    tom = originalTom;
+  }
+  return (
+    <FormattedMessage
+      id="Beregningsgrunnlag.BeregningTable.Periode"
+      key={`fom-tom${fom}${tom}`}
+      values={{
+        fom: moment(fom).format(DDMMYYYY_DATE_FORMAT),
+        tom: tom ? moment(tom).format(DDMMYYYY_DATE_FORMAT) : '',
+      }}
+    />
+  );
+};
+
+const statuserDetErSøktOmIPerioden = (bgPeriode, ytelsegrunnlag) => {
+  const fom = bgPeriode.beregningsgrunnlagPeriodeFom;
+  const tom = bgPeriode.beregningsgrunnlagPeriodeTom;
+  const perioder = ytelsegrunnlag.perioderSøktFor;
+  return perioder
+    ? perioder.filter(periode => !moment(fom).isBefore(periode.fom) && !moment(tom).isAfter(periode.tom))
+    : [];
+};
+
+const lagBeskrivelseMedBeløpRad = (tekstId, beløp) => {
+  return (
+    <Row>
+      <Column xs="10">
+        <FormattedMessage id={tekstId} />
+      </Column>
+      <Column xs="2">
+        <Normaltekst>{formatCurrencyNoKr(beløp)}</Normaltekst>
+      </Column>
+    </Row>
+  );
+};
+
+const lagRedusertBGRad = (tekstIdRedusert, beløpÅRedusere, tekstIdLøpende, løpendeBeløp) => {
+  const redusert = beløpÅRedusere * 0.8;
+  return (
+    <>
+      <Row>
+        <Column xs="10">
+          <FormattedMessage id={tekstIdRedusert} />
+        </Column>
+        <Column xs="2">
+          <Normaltekst>{formatCurrencyNoKr(redusert)}</Normaltekst>
+        </Column>
+      </Row>
+      {(løpendeBeløp || løpendeBeløp === 0) && (
+        <Row>
+          <Column xs="10">
+            <FormattedMessage id={tekstIdLøpende} />
+          </Column>
+          <Column xs="2">
+            <Normaltekst>{formatCurrencyNoKr(løpendeBeløp)}</Normaltekst>
+          </Column>
+        </Row>
+      )}
+    </>
+  );
+};
+
+const finnBeregningsgrunnlag = (bgperiode, statuserDetErSøktOm, status, inntektstak) => {
```

Comment:
Må ikkje denne ta hensyn til prioriteringsregler om det er søkt for både FL og SN?